ompi/mca/pml/ob1/pml_ob1_sendreq.c

```diff
@@ -448,6 +449,30 @@ int mca_pml_ob1_send_request_start_buffered(
 
     /* send */
     rc = mca_bml_base_send(bml_btl, des, MCA_PML_OB1_HDR_TYPE_RNDV);
+
+#ifdef SOFTWARE_EVENTS_ENABLE
+    volatile int64_t bytes_sent;
+    unsigned int i;
+    if(attached_event[OMPI_BYTES_SENT_USER] == 1){
+        if(sendreq->req_send.req_base.req_tag >= 0){
+            bytes_sent = 0;
+            for(i = 0; i < des->des_segment_count; i++){
```

Comment:
Correction: There are fewer bytes sent when I change all of them to bytes_delivered.  Working on a solution...